### ISSUE TITLE
Decoder is used as the generator, how to train the network?

### ISSUE THREAD

**Issue by 13462877152** [junyanz/pytorch-CycleGAN-and-pix2pix#1123]
Thank you very much for your excellent paper and code .  If the encoder is pretrained and fixed, only the decoder is trained as the generator. There are some skip-connection between encoder and decoder.  I follow the method used in Pix2Pix,  However, the loss of discriminator is about 0.25, and it dosen't reach equilibrium.  Can you give me some advice about traning this task.  Is there any research or paper like this ?  Thank you very much and looking forward your help.

**Comment by junyanz** [comment]
0.25 D loss looks normal to me. Some recent [work](https://arxiv.org/pdf/2008.00951.pdf) used a pre-trained network as part of image-to-image translation networks. You may want to have a look.

**Comment by 13462877152** [comment]
Thank you very much for your help, I learn from this paper. lsgan is used in my project like the pix2pix.  I am also puzzled when I should stop the training process.

**Comment by junyanz** [comment]
Hard to know when to stop training ML models. I guess until you get reasonable results.
